test/index.test.ts

```diff
@@ -1,92 +1,108 @@
-import nock from 'nock'
-import approvemanApp from '../src'
-import { Probot } from 'probot'
-import prOpenedPayload from './fixtures/basic/pr.opened.json'
-import prReopenedPayload from './fixtures/basic/pr.reopened.json'
-import prSynchronizePayload from './fixtures/basic/pr.synchronize.json'
-import { setConfigToBasic, setConfigNotFound } from './utils/config'
-import { checkApproved, setSinglePreviousReview, verifyReviewDismissed } from './utils/review'
-const fs = require('fs')
-const path = require('path')
+import nock from "nock";
+import approvemanApp from "../src";
+import { Probot } from "probot";
+import prGeneric from "./fixtures/basic/pr.json";
+import { setConfigToBasic, setConfigNotFound } from "./utils/config";
+import { checkSuccessStatus } from "./utils/status";
+import {
+  checkApproved,
+  setSinglePreviousReview,
+  verifyReviewDismissed,
+} from "./utils/review";
+import fs from "fs";
+import path from "path";
+import { StatusCodes } from "http-status-codes";
 
-jest.setTimeout(30000)
+jest.setTimeout(30000);
```

Comment:
Codacy found an issue: ['jest' is not defined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)